### ISSUE TITLE
[Feature] Get App Icon

### ISSUE THREAD

**Issue by 0PandaDEV** [HuakunShen/applications-rs#6]
It would be very useful to have a method to get the app icon/logo.

**Comment by HuakunShen** [comment]
Yes, I had implemented icon retrieval function in another project https://github.com/kunkunsh/kunkun that depends on this one. 
I am considering moving it to this project.
